app/controllers/users/sessions_controller.rb

```diff
@@ -8,7 +8,7 @@ class Users::SessionsController < Devise::SessionsController
 
   # POST /resource/sign_in
   # def create
-  #   super
+  #   super { |resource| @resource = resource }
```

Comment:
- [ ] Impossible to log in after signing up due to some error you can see in the image below.
<img width="1440" alt="Screenshot 2022-12-10 at 12 35 15 PM" src="https://user-images.githubusercontent.com/40503704/206853311-7c7310f4-eea8-4103-8e5e-47a9054e58da.png">